cartridges/openshift-origin-cartridge-python/usr/versions/2.7-scl/lib/update-configuration

```diff
@@ -8,19 +8,20 @@ function update-configuration {
     # SCL installs of python 2.7
     sclpath=$(dirname $(scl enable python27 "which python"))
     env_dir="${OPENSHIFT_PYTHON_DIR}/env"
```

Comment:
env_dir itself has a slash in there after OPENSHIFT_PYTHON_DIR - doesn't this need to be removed as well?